propagation/propagation.go

```diff
@@ -58,11 +79,16 @@ func (c MapCarrier) Keys() []string {
 // HeaderCarrier adapts http.Header to satisfy the TextMapCarrier interface.
```

Comment:
It would be nice to also document that it satisfies the  `MultiGetter` interface.

Reply:
I changed it to `// HeaderCarrier adapts http.Header to satisfy the MultiTextMapCarrier interface.` which should encompass all of the functionality.